util/distro/distro.go

```diff
@@ -62,3 +74,39 @@ func R() *DistributionResource {
 	}
 	return r.(*DistributionResource)
 }
+
+func StringsRes() (distroStringsRes DistributionResource) {
```

Comment:
As a utility, it should be as much common as possible, to serve the different needs of different environments, and different dashboard variants. In this background, I suggest you to make this function like this:

```go
func ReadResourceStrings(filePath string) (DistributionResource, error) {
    // ...
}
```

Reply:
Make sense!